perun-auditlogger/src/main/java/cz/metacentrum/perun/auditlogger/main/AuditLoggerStarter.java

```diff
@@ -0,0 +1,82 @@
+package cz.metacentrum.perun.auditlogger.main;
+
+import cz.metacentrum.perun.auditlogger.service.AuditLoggerManager;
+import cz.metacentrum.perun.core.api.ExtSourcesManager;
+import cz.metacentrum.perun.core.api.Perun;
+import cz.metacentrum.perun.core.api.PerunPrincipal;
+import cz.metacentrum.perun.core.bl.PerunBl;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.support.AbstractApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class AuditLoggerStarter {
+	private final static Logger log = LoggerFactory.getLogger(AuditLoggerStarter.class);
+
+	private AuditLoggerManager auditLoggerManager;
+	private AbstractApplicationContext springCtx;
+	private PerunPrincipal perunPrincipal;
+	private Perun perunBl;
+
+	public AuditLoggerStarter() {
+		this.perunPrincipal = new PerunPrincipal("perunLdapc", ExtSourcesManager.EXTSOURCE_NAME_INTERNAL, ExtSourcesManager.EXTSOURCE_INTERNAL);
```

Comment:
I would use own name instead of ``perunLdapc``.

Reply:
Fixed. Requires change to the perun.properties.

Reply:
Although, in light of the last change, probably not.

Reply:
As you changed usage of entry to bl, it shouldn't. If you wish, you can update ``defaultCoreProperties`` in ``perun-base.xml``.